### PR TITLE
'Fixed' highly-occurring background crash on CarPlay by disabling background snapshots altogether

### PR DESCRIPTION
**Pull request by Svantulden**
## Description

See title

### CODE REVIEW

File: platform/darwin/src/reachability.m [hunk 1]
```diff
@@ -465,9 +465,9 @@ -(void)reachabilityChanged:(SCNetworkReachabilityFlags)flags
 
 - (NSString *) description
 {
-    NSString *description = [NSString stringWithFormat:@"<%@: %#x (%@)>",
-                             NSStringFromClass([self class]), (unsigned int) self, [self currentReachabilityFlags]];
-    return description;
+    //NSString *description = [NSString stringWithFormat:@"<%@: %#x (%@)>",
+                             //NSStringFromClass([self class]), (unsigned int) self, [self currentReachabilityFlags]];
+    return @"Reachability";
```
Comment (Svantulden) [inline platform/darwin/src/reachability.m:470]: Idk why this is now needed 🤔. I've already built for Xcode 13 before and didn't get this compile error, but now I did:
```
❌  /Users/sander/Documents/GitHub/mapbox-gl-native/platform/darwin/src/reachability.m:469:63: cast to smaller integer type 'unsigned int' from 'MGLReachability *' [-Werror,-Wpointer-to-int-cast]

                             NSStringFromClass([self class]), (unsigned int) self, [self currentReachabilityFlags]];
```

Comment (MaartenZonneveld) [reply platform/darwin/src/reachability.m:470]: Gekke shit

File: platform/ios/src/MGLMapView.mm [hunk 2]
```diff
@@ -6293,7 +6293,8 @@ - (void)mapViewDidBecomeIdle {
         return;
     }
     
-    [self queueBackgroundSnapshot];
+    // TODO: Make a real fix for the crash, like https://github.com/maplibre/maplibre-gl-native/pull/68, but be careful, this brings other sporadic crashes!
+    //[self queueBackgroundSnapshot];
```
Comment (MaartenZonneveld) [inline platform/ios/src/MGLMapView.mm:6297]: Is dit de enige plek waar dit wordt aangeroepen?

**Review by MaartenZonneveld — APPROVED** [review]

File: platform/ios/src/MGLMapView.mm (hunk 2 above)
Comment (Svantulden) [reply platform/ios/src/MGLMapView.mm:6297]: Yes

**Review by bartvdput — APPROVED** [review]

**Pull request merged**